### DISCUSSION TITLE
[打假]拼多多2024津贴

### DISCUSSION THREAD

**Discussion by wangweilin20100603** [share121/inter-knot#2447] [其他]
![pdd](https://github.com/user-attachments/assets/d2b5cd35-d312-47d0-aeaa-88105c0b68bb)
小伙伴们，今天我们来打假拼多多这个2024津贴，看到有很多朱波都靠他领到了2024元，这肯定是假的，今天我就来打假。我们就按照视频的步骤来，先这样，然后再那样，最后在这样。哇！小伙伴们，我真的领导了2024津贴，原来是真的，大家快去领取吧！
![pdd](https://github.com/user-attachments/assets/b4e8e8e9-23ab-4989-834c-4d931f2350f0)

**Comment by wangweilin20100603** [comment]
我只是个玩抽象的小人机，别举报我

**Comment by Random114** [comment]
是真的，我领到了宇智波佐助（九喇嘛连结）

**Comment by yileinafs** [comment]
哇真的领导力，孩子很爱吃快去试试吧！

**Comment by linexzhou** [comment]
哇 敏感肌也能吃 强烈推荐<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/254352b8702fbdfe4b51919568b16f23_2308445851003216507.gif" alt="爱莉希雅-吃雪糕" width="75" >

**Comment by shan-dgha** [comment]
该用户默认好评

**Comment by Leo2126** [comment]
事真的，我领到了一只嗷呜

**Comment by CJHXAMSTLP** [comment]
是真的，我领到了阿姆斯特朗回旋加速喷气式阿姆斯特朗炮

**Comment by 1ctrl-cv** [comment]
这个委托我接了

**Reply by Bahajsjsh** [reply]
以读乱回是吧

**Comment by hanshui2077** [comment]
是真的，我领到了一直只因

**Comment by HyperMuteki0347** [comment]
本来我也不相信的，试了一下真的领到了！现在已经清空双倍等5.0更新了！<img src="https://bbs-static.miyoushe.com/static/2024/02/06/0b89a2c8791dd0111c409bc9d4b3d4c6_9065100843449507760.png" alt="闲云-自信仙鸟" width="75" >

**Comment by Genshin-Neuvillette** [comment]
我领到了一个空降旅，不骗人，他们现在正准备空降到我家

**Comment by Bahajsjsh** [comment]
是真的我领到了六道铠［秽土转生］

**Comment by dongfangxiaoqi** [comment]
为什么我只领到50
![mmexport1723903248389](https://github.com/user-attachments/assets/8cc0669a-0e7e-4983-b6f1-9777ab86fb12)

**Comment by xing-qiong** [comment]
是真的，我领到了赛罗奥特曼【贝利亚显现】<img src="https://img-static.mihoyo.com/communityweb/upload/408018316100a8a8a846c68acedaa191.gif" alt="米游姬 卖萌" width="75" >

**Comment by Nie7bai** [comment]
真的，我刚刚领了1600菲林，我发视频了，你们快来看![居然真的能领！]（https://www.bilibili.com/video/BV1bi421h79s/?spm_id_from=333.788.recommend_more_video.-1&vd_source=f4b3a2e28f73115f67c5b50438cd9cf6）<img src="https://bbs-static.miyoushe.com/static/2024/07/31/fdc5e0e30e577fb208f29effa93510ed_8442612635207222552.png" alt="云璃 期待" width="75" >

**Reply by I-ikun** [reply]
每天上一当，当当不一样<img src="https://img-static.mihoyo.com/communityweb/upload/fcb0d849f766bb544445a7ac0d5b9677.png" alt="妮可_生气" width="75" >

**Reply by NZTWR** [reply]
我就领到两分钱

**Comment by huner006** [comment]
是真的，我领到了6480！不多说了，我先去氪几单了！
